flink-table/flink-sql-gateway/pom.xml

```diff
@@ -98,6 +98,13 @@
             <artifactId>okhttp</artifactId>
             <scope>test</scope>
         </dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-runtime</artifactId>
+			<version>${project.version}</version>
+			<type>test-jar</type>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Not sure if we need this. I think currently `flink-sql-gateway` already (indirectly) depends on `flink-runtime`.

Reply:
Yes. I noticed that some of the dependencies of SQL gateway have already relied on the test jar of the runtime module. This has been removed.